CHANGELOG.rst

```diff
@@ -1,3 +1,9 @@
+1.8.0 (2020-01-10)
+------------------
+
+* Add new category: "standard volume per standard volume"
+* Move unit ``sm3/sm3`` from "volume per volume" to "standard volume per standard volume"
```

Comment:
Let's hope no one was using this unit with the former quantity =)

Reply:
Yep, but we believe only ALFAsim was using.